css/styles.css

```diff
@@ -0,0 +1,399 @@
+/* font pack*/
+
+    @import url('https://fonts.googleapis.com/css2?family=Bakbak+One&family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap');
+
+    @import url('https://fonts.googleapis.com/css2?family=Bakbak+One&family=Open+Sans:ital,wght@0,300..800;1,300..800&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+    @import url('https://fonts.googleapis.com/css2?family=Bakbak+One&family=Inter:wght@100..900&family=Open+Sans:ital,wght@0,300..800;1,300..800&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+/****************************************************************/
+
+html {
+    background-color: #0D0D11;
+}
+
+
+h1,h2,h3 {    
+    font-family: Bakbak one, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    color: #fff;
+}
+
+
+p {
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    color: #FFFFFF;
+}
+
+body {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+.hidden {
+    display: none;
+}
+
+/*colored text in the headers*/
+
+
+.text-label, .colored-text {
+    color: #21e786;
+}
+
+/*header*/
+
+.main-header {
+    background-color:#040b11;
+        width: 1920px;
+        height: 108px;
```

Comment:
I think I understand why you did it and this is an interesting train of thought. As part of this task, we don't need to set heights and widths, let's remove them so that we don't get confused in the future? And a little secret, then we will have a container that will adjust the width